modules/opensearch/opensearch.client.inc

```diff
@@ -105,7 +112,10 @@ function opensearch_get_objects(array $ids, $local = FALSE) {
     else {
       $request->setObjectIds($chunk);
     }
-    $objects += opensearch_execute($request);
+    $result = opensearch_execute($request);
+    if (is_array($result)) {
```

Comment:
I do not understand the check here - and perhaps not the previous state of the code either. According to the DocBlock `opensearch_execute` returns `\TingClientSearchResult|bool`. This change is based on the option that the function will return an array.

If the DocBlock is wrong then it should be updated accordingly.

Reply:
It either returns an array or NULL. Will update DocBlock.

Reply:
@kasperg I have updated the docblock to describe exactly what is being returned based on the type of request passed to opensearch_execute().

Reply:
Thanks. Much appreciated.